test/test_pyrobot.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python3
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+import sys
+
+import numpy as np
+
+import mock
+from habitat.config.default import get_config
+from habitat.sims import make_sim
+
+
+class CameraMock:
```

Comment:
As discussed offline, it will be better to mock the builder function itself to return mock of the class you are testing. This would ensure that you never have to mock anything on the PyRobot side.